config/sidebar.velocity.ts

```diff
@@ -108,7 +108,7 @@ const velocity: SidebarsConfig = {
             "dev/api/event",
             "dev/api/command",
             "dev/api/scheduler",
-            "dev/how-to/plugin-messaging",
+            "dev/api/plugin-messaging",
```

Comment:
this might need a redirect?

Reply:
Nope, that's the path on disk. The slug (at top of file) is unchanged